src/glfw-sandbox/engine.h

```diff
@@ -0,0 +1,96 @@
+#pragma once
+#include <windows.h>
+#include <GL/gl.h>
+#include <GL/glu.h>
+
+#define GLFW_INCLUDE_NONE
+#include <GLFW/glfw3.h> // Will give use GLFW + GL
+
+#include <iostream>
+
+
+class windowCreator
+{
+private:
+	int resWidth{};
+	int resHeight{};
+	
+	// Could you please clarify what these three do?
+	// Are these ponters? If posssible just a little clarification how this works
+	// This might be tied to main.cpp line 23-32
+	char *title;
+	GLFWmonitor *monitor;
+	GLFWwindow *share;
+
+	int fieldOfView{};
+
+
+public:
+
+	windowCreator(int resW, int resH, int fov) // for now leave other values and hardcode like a noob, Petr help <3 :D
+		: resWidth{ resW }, resHeight{ resH }, fieldOfView{fov}
+	{	
+		if (!glfwInit())
+		{
+			std::cout << "Failed to initialize GLFW library" << std::endl;
+			//return 1; Why I can't return 1 here?
```

Comment:
...because this is a constructor. You can, still raise an exception, in theory, on the other side, it is not recommended to fire exceptions from constructor. We are trapped! :)